storage/entities/peer.js

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const ft = require('../utils/filter_types');
+const { stringToByte } = require('../utils/inputSerializers');
+const { NonSupportedFilterTypeError } = require('../errors');
+const BaseEntity = require('./base_entity');
+
+const defaultCreateValues = {};
+
+const readOnlyFields = [];
+
+/**
+ * Basic Peer
+ * @typedef {Object} BasicPeer
+ * @property {number} id
+ * @property {inet} ip
+ * @property {number} wsPort
+ * @property {number} state
+ * @property {string} os
+ * @property {string} version
+ * @property {number} clock
+ * @property {string} broadhash
+ * @property {number} height
+ */
+
+/**
+ * Peer Filters
+ * @typedef {Object} filters.Peer
```

Comment:
Based on the filters you added, please complete this `typedef`

Reply:
Done